specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2025-02-01-preview/appconfiguration.json

```diff
@@ -2846,19 +3048,84 @@
         }
       }
     },
+    "ExperimentationListResult": {
+      "description": "The result of a request to list experimentations.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Experimentation"
+          }
+        },
+        "nextLink": {
+          "description": "The URI that can be used to request the next set of paged results.",
+          "type": "string"
+        }
+      }
+    },
+    "Experimentation": {
+      "description": "The experimentation resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
```

Comment:
Can you put a blurb in the description describing the requirement on the name that we discussed. I believe we agreed the name must be "Experimentation"

Reply:
Oh, I see you created ExperimentationNameParameter. That should suffice then.